packages/prelude/src/function.js

```diff
@@ -47,3 +47,9 @@ export function curry4 (f) {
   }
   return curried
 }
+
+// noop :: () -> undefined
+export const noop = () => undefined
+
+// bind :: (a -> b) -> c -> (d -> e)
+export const bind = (fn, object) => fn.bind(object)
```

Comment:
Before we add this, I'd like to confirm that `bind`'s performance across VMs has improved.  Previously, it has been a significant performance hit, both when calling `fn.bind` and when calling previously-bound functions.

Reply:
Does performance here really matter? After all, it’s just a convenience API and not used internally.

Reply:
I think it does matter.  The mostjs code never uses bind because of its history of performance issues--the current code just creates lambdas in spots it absolutely needs to.  If we add a `bind` function to prelude (and prelude is primarily for use in mostjs code), the intent is clearly that we will use it.  If that `bind` function also has performance issues, we could be introducing performance issues.